guides/common/modules/proc_creating-a-remote-file-repository.adoc

```diff
@@ -25,24 +25,40 @@ endif::[]
 
 To create a file type repository in a remote directory, complete the following procedure:
 
-. On your remote server, ensure that the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. On your remote server, ensure that the Server and {project-client-name} repositories are enabled.
```

Comment:
Should we change `Server` here to one of the EL macros? Server can be confusing especially given RHEL 8 calls it Base OS and AppStream  @ekohl

Reply:
Perhaps not even specify the names at all. Just say:
```suggestion
. On your machine, ensure that the right repositories are enabled.
```

This is because utils could in theory also run from a desktop (at least with EL8).